### PR TITLE
Integrate into privileged xpi signing pipline

### PR DESCRIPTION
**Pull request by GregSherman**
Changes a few things to get the build right.

- Icon was not square (this is temporary until we get a real icon)
- Was building .zip not .xpi
- renamed xpi to firefox-bridge as expected

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Community Participation Guidelines
+
+This repository is governed by Mozilla's code of conduct and etiquette guidelines.
+For more details, please read the
+[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
```
Comment (willdurand) [inline CODE_OF_CONDUCT.md:5]: Make it a single line please:

```suggestion
This repository is governed by Mozilla's code of conduct and etiquette guidelines. For more details, please read the [Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Community Participation Guidelines
```
Comment (willdurand) [inline CODE_OF_CONDUCT.md:1]: Maybe we could put this file in a `.github/` folder.

File: package.json [hunk 3]
```diff
@@ -35,9 +35,9 @@
   "author": "mozilla",
   "license": "ISC",
```
Comment (willdurand) [inline package.json:36]: That shouldn't be `ISC` but `MPL-2.0`.

File: package.json [hunk 4]
```diff
@@ -35,9 +35,9 @@
   "author": "mozilla",
```
Comment (willdurand) [inline package.json:35]: s/mozilla/Mozilla

File: package.json [hunk 5]
```diff
@@ -24,7 +24,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mozilla/firefox-launch.git"
+    "url": "git+https://github.com/mozilla-extensions/firefox-bridge"
   },
   "keywords": [
```
Comment (willdurand) [inline package.json:29]: You can probably omit the keywords.

Comment (willdurand) [reply package.json:29]: Since I cannot comment on all lines, is the `directories` field required? If not, please remove it.

Same question, for `main`.

Comment (willdurand) [reply package.json:29]: The `description` should also be fixed:

```
Extension that enables the user to launch Firefox from another browser
```

**Review by willdurand — CHANGES_REQUESTED** [review]
Looking good, thanks - I am requesting some changes before we can land this PR.

File: src/firefox/manifest.json [hunk 6]
```diff
@@ -18,7 +19,7 @@
   },
   "browser_action": {
     "default_icon": {
-      "32": "images/firefox-launch/32grey.png"
```
Comment (willdurand) [inline src/firefox/manifest.json:21]: I am wondering if we should update the add-on ID in `bss.gecko.id`, WDYT? It is currently "firefox-launch@mozilla.com" but we started. to rename everything to "firefox-bridge".

Comment (GregSherman) [reply src/firefox/manifest.json:21]: No objections, was just a placeholder I overlooked.

File: package.json [hunk 7]
```diff
@@ -24,20 +20,14 @@
...
   },
-  "keywords": [
-    "firefox",
-    "addon",
-    "chrome",
-    "edge"
-  ],
-  "author": "mozilla",
-  "license": "ISC",
+  "author": "s/mozilla/Mozilla",
```
Comment (willdurand) [inline package.json:25]:
```suggestion
  "author": "Mozilla",
```

Sorry for the confusion, "s/<old>/<new>" is an ancient term I use to say "replace <old> with <new>".

**Review by willdurand — APPROVED** [review]
Thanks. Please address the only comment I left before merging.

**Pull request merged**